app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,6 @@
   <head>
     <title>Quibble</title>
     <%= stylesheet_link_tag    "application", media: "all" %>
-    <%= stylesheet_link_tag    "materialize", media: "screen, projection" %>
```

Comment:
You'll notice that it breaks more subtle things, like not using the Roboto font that comes with Materialize (which isn't that big of a deal), but it also breaks our mobile friendly sidebar. So if a user is viewing this on their phone, they would never be able to sign out since that button only appears in the sidebar, which is invisible now.